### PR TITLE
Fix path for entities which were contributed via UI

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/44113

## Description
Added a fix to make sure indicator fields and mappers are in the correct path.

## Screenshots
Screen Shot 2021-12-09 at 14 40 40

Screen Shot 2021-12-09 at 15 50 20

Screen Shot 2021-12-09 at 15 51 23

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:256]: Unless we have a good reason
```suggestion
        else:
```

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:262]: please add docstring with elaborate details

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:273]: same here

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:250]: same here

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:275]: let's give more descriptive names than `key` and `value`

File: demisto_sdk/commands/init/contribution_converter.py
Comment (guykeller) [inline demisto_sdk/commands/init/contribution_converter.py:280]: This name can be easily confused with `dst_path`, let's make sure the naming reflects the content

**Review by guykeller — APPROVED** [review]
Mostly stylistic suggestions, looks good other than that.
I feel strongly about the `else` clause and the docstrings, please fix them before merging.
The rest is for your consideration and judgement

File: demisto_sdk/commands/init/contribution_converter.py
Comment (barchen1) [reply demisto_sdk/commands/init/contribution_converter.py:256]: In case the base name is classifier and the file is really a classifier we will get None from `fix_dst_file_path_for_mapper` func, so we need this condition to make sure if it is None or not.

File: demisto_sdk/commands/init/contribution_converter.py
Comment (barchen1) [reply demisto_sdk/commands/init/contribution_converter.py:262]: done

File: demisto_sdk/commands/init/contribution_converter.py
Comment (barchen1) [reply demisto_sdk/commands/init/contribution_converter.py:283]: done

File: demisto_sdk/commands/init/contribution_converter.py
Comment (barchen1) [reply demisto_sdk/commands/init/contribution_converter.py]: done

**Comment by barchen1** [comment]
Waiting for this PR to get merged https://github.com/demisto/contribution-management/pull/160, cause the fix for https://github.com/demisto/etc/issues/44113 is also in there.

**Pull request closed without merging**
